conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/ApiFilterQueryString.scala

```diff
@@ -0,0 +1,21 @@
+package tech.cryptonomic.conseil.api.routes.platform.data
+
+import cats.Functor
+import endpoints.algebra
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiFilter.Sorting
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiFilter.Sorting._
+
+trait ApiFilterQueryString { self: algebra.JsonEntities =>
```

Comment:
Maybe docs would be helpful to make sense of all our bandwagon of traits that are mixed and meshed?

Personally I have a hard time following the logic of all the pieces when I read the code without having to put hands on it....

Reply:
Sure. I'll provide some documentation where possible and sensible :)